CMakeLists.txt

```diff
@@ -287,14 +287,6 @@ if(Threads_FOUND AND CMAKE_USE_PTHREADS_INIT)
     "${CMAKE_REQUIRED_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT}")
 endif()
 
-include(CheckCSourceCompiles)
-if(MSVC)
-  set(CMAKE_REQUIRED_FLAGS "${CMAKE_C_FLAGS} /WX /W4")
-else()
-  set(CMAKE_REQUIRED_LIBRARIES m)
```

Comment:
we should probably keep ``set(CMAKE_REQUIRED_LIBRARIES m)``

Reply:
So the whole block isn't there just for the now-removed check_c_source_compiles?

Reply:
hum, I see there were past math function detection in 2622d38515, but they have gone to now. So, probably disregard my comment and this is indeed no longer needed